st2common/tests/unit/test_query_base.py

```diff
@@ -40,9 +40,9 @@ def test_fire_queries_doesnt_loop(self):
         mock_query_state_1 = QueryContext(
             uuid.uuid4().hex,
             uuid.uuid4().hex,
-            'mistral_v2',
+            'orquesta_runner',
```

Comment:
The orquesta runner doesn't use the Querier. Mistral is the only one that uses the querier.  I want to preserve the generic solution here.  Can we add a mock async runner in the test setup and use that here instead?

Reply:
I'll look into getting that mock runner in this week, and update tests to use the mock runner.